### ISSUE TITLE
Format times to hours and minutes

### ISSUE THREAD

**Issue by yuki-93** [Jugendhackt/fahrplanplusplus#15]
As shown in the presentation, there are times which are displayed as `1800seconds`. 

Consider formatting values to handy hours and minutes, like `1h30m`.

**Comment by margau** [comment]
Internally, it is easier to calculate with seconds

**Comment by ghost** [comment]
Yes for internal this is fine. This issue is meant for the displaying times for the user.
